### ISSUE TITLE
Screan Backlight

### ISSUE THREAD

**Issue by N9CFG** [Tunas1337/UV-K5-Modded-Firmwares#17]
Any chance to get longer backlight times? Like 10 seconds or more.
Thanks, 
N9CFG

**Comment by Tunas1337** [comment]
Absolutely! That is an achievable thing. I'll get back to you as soon as there is progress on it.
In the meantime, check out Adam Mnemonic's [very cool "mod kitchen"](https://github.com/amnemonic/Quansheng_UV-K5_Firmware/tree/main/uvmod_kitchen) for other modifications that you might like, that haven't been built into firmwares here.

**Comment by marusc1** [comment]
An option to leave it permanently on would be grrat for working with external power connected via USB or battery contacts.
